Sources/StreamChatSwiftUI/ChatChannel/Reactions/ReactionsOverlayView.swift

```diff
@@ -108,7 +116,7 @@ public struct ReactionsOverlayView<Factory: ViewFactory>: View {
                 .offset(
                     x: messageActionsOriginX
                 )
-                .padding(.top, 16)
+                .padding(.top, padding)
```

Comment:
[Nitpicky] We could think about renaming the `padding` variable to something like `paddingAmount` so that it doesn't interefere with the modifier naming-wise.

Reply:
Good point, I renamed it to `paddingValue` - I think that's better.